css/main.css

```diff
@@ -440,17 +456,23 @@ textarea {
     justify-content: center;
   }
 
-  .menu-bars {
+  .container {
     display: none;
   }
 
-  #desktop-nav {
-    display: block;
-  }
-
   #desktop-nav ul {
```

Comment:
- [ ] You could add an additional class to the element and specify it in your CSS instead of ids 🤔 !!

_CSS classes can do everything IDs can._ 💯 

_IDs have a much higher specificity than classes. A class on its own can't overwrite styles belonging to an ID. To "beat" the ID, you would need either more IDs or to use ! important , which can begin specificity wars in your stylesheets._

For more info please refeer to [THIS](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) .